lib/config/configItems.joi.js

```diff
@@ -5,7 +5,7 @@ const constants = require('../../extensions/replication/constants');
 const {
     authTypeAccount,
     authTypeService,
-    authTypeAssumeRole,
+    authTypeAssumeRole
 } = require('../constants');
```

Comment:
not having comma does not follow Scality guidelines https://github.com/scality/Guidelines/blob/development/8.2/index.js#L19C10-L19C22

It will be enforced anyway when we upgrade the dependencies...